### PR TITLE
Feature/comment

### PR DESCRIPTION
**Pull request by Bwananah**
Add comments to posts, this branch was created from Ahmed's upvote and downvote feature branch.
The comments aren't linked in the code, but are connected in the database using the correct poiKey and postId in the comments path.

I also fixed some duplication issues with RecyclerAdapters

Files changed  (I hate GitHub):
comment.xml
activity_post.xml
Comment.kt
PostActivity.kt
ForumActivity.kt
PostActivityTest.kt
ForumActivityTest.kt
CommentsRecyclerAdapter.kt
ForumRecyclerAdapter.kt
FriendsListRecyclerAdapter.kt
ReviewsRecyclerAdapter.kt
MeiliRecyclerAdapter.kt

### CODE REVIEW

File: app/src/main/java/com/github/epfl/meili/forum/CommentsRecyclerAdapter.kt [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+        )
+
+    override fun onBindViewHolder(holder: RecyclerView.ViewHolder, position: Int) =
+        (holder as CommentViewHolder).bind(items[position], userId)
+
+    class CommentViewHolder(itemView: View, private val viewModel: MeiliViewModel<Comment>) :
+        RecyclerView.ViewHolder(itemView) {
+        private val author: TextView = itemView.findViewById(R.id.comment_author)
+        private val text: TextView = itemView.findViewById(R.id.comment_text)
+        private val commentId: TextView = itemView.findViewById(R.id.comment_id)
```
Comment (aman3014) [inline app/src/main/java/com/github/epfl/meili/forum/CommentsRecyclerAdapter.kt:30]: What is the commentId used for?

**Review by aman3014 — COMMENTED** [review]
Looks awesome! Just one question before I approve

**Review by aman3014 — APPROVED** [review]
Great job on this feature! Perfect coverage and very clean code 🎉 
Thanks for modularising the RecyclerAdapters too!

**Review by MarcelTorne — APPROVED** [review]
Thanks for all your work Ewan! Really cool feature and awesome refactoring! Thanks!!

**Pull request merged**